src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/Common.ps1

```diff
@@ -93,7 +93,7 @@ Cleans the created resource groups
 function Clean-ResourceGroup($rgname)
 {
     if ((Get-NetworkTestMode) -ne 'Playback') {
-        Remove-AzureRmResourceGroup -Name $rgname -Force
+        Remove-AzResourceGroup -Name $rgname -Force
```

Comment:
You don't want to do this - the resourcemanager script uses the old cmdlet names and aliases are turned on in all the tests